src/authentication/decode-abax-profile.ts

```diff
@@ -23,7 +23,13 @@ export const abaxIdTokenPayload = z.object({
   email_verified: z.boolean(),
   phone_number: z.string().optional(),
   'http://schemas.abax.no/identity/claims/securitystamp': z.string(),
-});
+}).transform(data => ({
+  atHash: data.at_hash,
+  authTime: data.auth_time,
+  emailVerified: data.email_verified,
+  phoneNumber: data.phone_number,
+})
```

Comment:
The return value of the tranform function is what is returned when parsing data with this schema. We want all the data here, not just these four fields. This applies to many other examples throughout this PR.